src/components/counter/counter.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './counter.css';
+class Counter extends React.Component {
+    constructor(props){  
+        super(props)    
+        this.state = {count : 0}  
+        this.incrementCount = this.incrementCount.bind(this)
+        this.decrementCount = this.decrementCount.bind(this) 
+      }  
+        
+      incrementCount(){  
+        let i=this.state.count;
+        i++;
+        this.setState({count : i})  
+      } 
+      decrementCount(){  
+        let i=this.state.count;
+        i--;
+        this.setState({count : i})  
+      }  
+    render() {
+      return (
+        <div className="counter-container">
+        <button className="decrement" onClick={this.decrementCount}>-</button>
```

Comment:
Please use proper naming convention for className and methods.
Follow https://www.linkedin.com/pulse/react-js-naming-convention-kristiyan-velkov/

Reply:
Thanks for this very useful reference. But do you see I need to make any changes here, as I am using the same convention for className and methods as suggested in the document.
![image](https://github.com/debasen/netfilx-roulette/assets/28741376/6a7cace8-637d-46c2-b71c-9c518a958a6a)

Reply:
You can use handleIncrement and handleDecrement instead.

Reply:
Fixed it